.gitignore

```diff
@@ -4,7 +4,7 @@
 /.grype.yaml
 
 CHANGELOG.md
-VERSION
+/VERSION
```

Comment:
One case insensitive file systems (such as macOS), this line previously resulted in ignoring files under `./grype/version/`.